components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/bean/RewriteContext.java

```diff
@@ -25,13 +25,16 @@
 
     private String context;
 
-    private Pattern pattern;
+    private Pattern tenantContextPattern;
+
```

Comment:
Unnecessary new line.

Reply:
This class has followed a pattern having spaces between fields